functions.php

```diff
@@ -481,7 +488,7 @@ function primer_scripts() {
 
 	wp_style_add_data( $stylesheet, 'rtl', 'replace' );
 
-	$nav_dependencies = ( is_front_page() && has_header_video() ) ? array( 'jquery', 'wp-custom-header' ) : array( 'jquery' );
+	$nav_dependencies = ( is_front_page() && ( function_exists( 'has_header_video' ) && has_header_video() ) ) ? array( 'jquery', 'wp-custom-header' ) : array( 'jquery' );
```

Comment:
The last two conditions don't have to be grouped, when the second one fails the third won't be attempted.

```php
is_front_page() && function_exists( 'has_header_video' ) && has_header_video()
```